shared/constants/selectors.js

```diff
@@ -11,8 +11,9 @@ const loggedInSelector = ({config: {loggedIn}}: TypedState) => loggedIn
 const cachedSearchResults = ({entities: {searchQueryToResult}}: TypedState, searchQuery: SearchQuery) =>
   searchQueryToResult.get(searchQuery)
 
-const searchResultSelector = ({entities: {searchResults}}: TypedState, username: string) =>
-  searchResults.get(username).toObject()
+const searchResultSelector = ({entities: {searchResults}}: TypedState, username: string) => {
+  return searchResults.get(username)
```

Comment:
We have to be careful here. this means you're passing a record down instead of an object. This makes the object slightly different but it can appear similar sometimes. In general we've been moving towards not passing immutable props down into dumb components (not that this selector is doing that, but just to mention it)